addons/website_slides/models/slide_slide.py

```diff
@@ -19,6 +19,16 @@
 from odoo.addons.http_routing.models.ir_http import url_for
 
 
+class SlideUsersRelation(models.Model):
+    _name = 'slide.users'
+    _description = 'Slide / Users decorated m2m'
+    _table = 'slide_users'
+
+    slide_id = fields.Many2one('slide.slide', index=True, required=True)
+    user_id = fields.Many2one('res.users', index=True, required=True)
+    vote = fields.Integer('Vote', default=0)
+
```

Comment:
Maybe we should add an SQL constraint to restrict value of vote to [-1, 0, 1]. What do you think ? :)